.travis.yml

```diff
@@ -58,6 +58,7 @@ script:
     - cargo build --locked --no-default-features --features "jackaudio-backend"
     - cargo build --locked --no-default-features --features "rodio-backend"
     - cargo build --locked --no-default-features --features "sdl-backend"
+    - cargo build --locked --no-default-features --features "gstreamer-backend"
```

Comment:
I think you'll also need to add some packages to `addons.apt.packages` where `libasound2-dev` and the like are set.

Probably `gstreamer1.0-dev` and `libgstreamer-plugins-base1.0-dev`

Reply:
Is it kosher to add apt dependencies to _every_ Travis build just because one optional feature requires it? I guess, looking at the existing stuff under addons/apt/packages, that's the case, but it seems a bit limiting to not be able to scope these to a feature. Ah, well.

Reply:
Yeah it doesn't seem ideal, but there's always a trade-off with time to run. If we just install all the apt packages together and then run all our builds in one container that's surely a lot faster than separately building each. Free travis is limited to 4 concurrent jobs I believe (we already have 5).

I this case we probably do want to move the gstreamer-backend feature build to it's own list of scripts to avoid trying on rust 1.33 - so dependencies could move with it into a 6th job I guess?

Reply:
It might be possible to scope the installation of packages to a feature with a bash script. Feel free to write one, but personally I hate dealing with CIs enough as it is without needing to spend extra time optimising them. If it builds in 10 mins or 30 mins I'm pretty indifferent, though as always open to someone doing it better.